bittensor/_cli/cli_impl.py

```diff
@@ -41,7 +41,8 @@ def __init__(self, config: 'bittensor.Config' ):
                 config (:obj:`bittensor.Config`, `required`): 
                     bittensor.cli.config()
         """
-        if not config.no_version_checking:
+        # (d)efaults to True if config.no_version_checking is not set.
+        if not config.get("no_version_checking", d=True):
```

Comment:
What is the default value currently?

Reply:
By default it may not be an attribute. The dot get accessor will default to `None` so the override makes it default `True`